docs/suh-template/plan/20260605_20260604_카카오_로그인_카카오앱_의존_제거_Firebase_Custom_Token_전환.md

```
@@ -0,0 +1,186 @@
+## 1. 요약
+
+카카오 로그인을 카카오톡 앱 설치 여부와 무관하게 동작하도록 **인앱 웹 로그인 폴백**을 추가하고,
+Firebase 인증 방식을 기존 OIDC Provider에서 **백엔드 발급 Custom Token** 방식으로 전환한다.
+두 변경은 `kakao_auth_service.dart` 하나와 신규 API 메서드 하나로 처리 가능하며, 로그인 이후 흐름(`signInWithSocial`)은 그대로 유지된다.
+
+---
+
+## 2. 배경 및 목적
+
+### 문제/필요성
+| 문제 | 내용 |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)